README.md

```diff
@@ -33,70 +33,106 @@ For OpenShift deployment clone [**OpenShift Ansible project**][openshift-ansible
 $ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
 ```
 
-## Install KubeVirt on existing OpenShift cluster
+## (Optional) Setup a cluster
 
-When you have your cluster up and running you can use
-[install-kubevirt-on-openshift.yml](./install-kubevirt-on-openshift.yml)
-playbook to install KubeVirt from given manifest.
+This section describes how to setup cluster on given machines. There are two
+playbook which you can use depends on what cluster you want to bring up.
 
-| Name             |  Value        | Description                            |
-| ---------------- | ------------- | -------------------------------------- |
-| kubevirt_mf      | string | Path to KubeVirt manifest, you can get one from [releases](https://github.com/kubevirt/kubevirt/releases) or build it from sources |
-| openshift_ansible_dir | string | Path to [OpenShift Ansible][openshift-ansible-project] repository |
-| kconfig | string | Path to kubeconfig |
 
-```bash
-$ wget https://github.com/kubevirt/kubevirt/releases/download/v0.2.0/kubevirt.yaml
-$ git clone https://github.com/openshift/openshift-ansible.git
-$ ansible-playbook -i localhost, --connection=local \
-        -e "openshift_ansible_dir=openshift-ansible/ \
-        kconfig=$HOME/.kube/config \
-        kubevirt_mf=kubevirt.yaml" \
-        install-kubevirt-on-openshift.yml
-```
+### General resources for cluster
 
-[![asciicast](https://asciinema.org/a/161278.png)](https://asciinema.org/a/161278)
+There are three key cluster components which needs to be deployed.
 
+* master - is component which is brain of cluster
```

Comment:
The phrasing is a bit off, maybe its just me :)
What do you think about using the definition from the K8S docs:

https://docs.openshift.com/container-platform/3.3/architecture/infrastructure_components/kubernetes_infrastructure.html

"The master is the host or hosts that contain the master components, including the API server, controller manager server, and etcd. The master manages nodes in its Kubernetes cluster and schedules pods to run on nodes."

Reply:
Done